### DISCUSSION TITLE
Looking for more use-cases using Actors.

### DISCUSSION THREAD

**Discussion by tawanorg** [statelyai/xstate#1224] [General]
Hi 

I love this library and want to implement this on one of my large application. 

Here is some examples but still could not get the xstate machine to communicate through React hooks between different actor. 

https://codesandbox.io/s/xstate-react-context-hooks-with-actors-tyzt7?file=/src/index.js

Any idea?

**Comment by davidkpiano** [comment]
@ttawan Can you please make this a more minimally reproducible example?
